nautobot/extras/templates/extras/secretsgroup_edit.html

```diff
@@ -53,13 +53,23 @@
                                             <!-- FORM-ERROR {{ field.name }}: {{ error }} -->
                                             <li clas="text-danger">{{ error }}</li>
                                         {% endfor %}
-                                    </ul>
-                                {% endif %}
-                            </td>
-                        {% endfor %}
-                    </tr>
-                {% endfor %}
-            </table>
+                                    {% endif %}
```

Comment:
Are the changes to this file correct/relevant? They look wrong to me. Maybe revert?